### PR TITLE
[BACKLOG-37109][BACKLOG-37110]-Tab Next and Previous utility method.

### PR DESCRIPTION
**Pull request by phalguni-a**
@pentaho/wcag , please review

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 19s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **378**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-857/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java [hunk 1]
```diff
@@ -283,6 +283,16 @@ public static native Element findPreviousKeyboardFocusableElement( Element elem
     return $wnd.pho.util._focus.previousTabbable(elem);
   }-*/;
 
+  public static void tabNext( Element rootElem ) {
+    Element elem = ElementUtils.findNextKeyboardFocusableElement( rootElem );
+    elem.focus();
```
Comment (dcleao) [inline widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java:288]: Although unlikely, must guard against `null` value.

File: widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java [hunk 2]
```diff
@@ -283,6 +283,16 @@ public static native Element findPreviousKeyboardFocusableElement( Element elem
...
+  }
+
+  public static void tabPrevious( Element rootElem ) {
+    Element elem = ElementUtils.findPreviousKeyboardFocusableElement( rootElem );
+    elem.focus();
+  }
```
Comment (dcleao) [inline widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java:294]: Idem.

**Review by dcleao — CHANGES_REQUESTED** [review]

File: widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java (hunk 1 above)
Comment (phalguni-a) [reply widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java]: addressed

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 54s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **378**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-857/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 54s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **378**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-857/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Review by hv-vamsi — APPROVED** [review]

**Pull request merged**